API/projectClass/projectClass.m

```diff
@@ -749,31 +1209,22 @@ function setLayersAndContrasts(obj, oldModel)
 
     end
 
-    methods (Hidden)
-
-        function domainsObj = domainsClass(obj)
```

Comment:
The reason the routine was written like this was to take advantage of converter methods: https://uk.mathworks.com/help/matlab/matlab_oop/converting-objects-to-another-class.html
I can't remember what specific feature we wanted at this point in time, but we should try and remember what it was and not throw it away rashly.